cellmlmanip/mathml2sympy/transpiler.py

```diff
@@ -0,0 +1,440 @@
+"""
+Parses Content MathML and returns equivalent SymPy expressions
+
+Content Markup specification: https://www.w3.org/TR/MathML2/chapter4.html
+
+TODO: Validate MathML input
```

Comment:
This would be a separate module, probably just validating against a (RELAX NG) schema. There's one in the old pycml.

Reply:
Removed note from transpiler